code/game/turfs/space/space.dm

```diff
@@ -15,8 +15,20 @@
 
 /turf/space/New()
 	. = ..()
+	if(!global_space_area)
+		global_space_area = new /area/space()
+		global_space_area.white_overlay = image(loc=global_space_area,icon='icons/turf/space.dmi',icon_state="white",layer=AREA_LAYER+0.1)
+	if(istype(loc,/area/space))
+		global_space_area.contents += src
+	else
+		var/area/A = loc
+		if(!A.white_overlay)
+			A.white_overlay = image(loc=A,icon='icons/turf/space.dmi',icon_state="white",layer=AREA_LAYER+0.1)

```

Comment:
This lacks any spacing in the image call. It should look like `image(loc = A, icon = 'icons/turf/space.dmi', icon_state = "white", layer = AREA_LAYER + 0.1)`